ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -210,6 +204,16 @@ class ShyHeaderController: UIViewController {
         view.bringSubviewToFront(paddingView)
     }
 
+    private func setupShyHeaderView() {
+        shyHeaderView.accessoryView = contentViewController.navigationItem.accessoryView
+        shyHeaderView.navigationBarShadow = contentViewController.navigationItem.navigationBarShadow
+        shyHeaderView.paddingView = paddingView
```

Comment:
it feels like all the paddingView logic should be in shyheaderView directly and not in shyheadercontroller? if the padding view is the a subview of shyheaderview? the ownership of the view seems really bad. I think maybe we need to have different solution for 0.14 release vs 0.13 release.